core/chaincode/chaincodetest.yaml

```diff
@@ -303,22 +304,23 @@ chaincode:
     pull: false
 
     golang:
-        # golang will never need more than baseos
+        # Compiled Go chaincodes will be copied to the fabric-baseos runtime image.
+        # TWO_DIGIT_VERSION represents Fabric major.minor version.
```

Comment:
why not do that for Golang too? The state of Golang chaincode is the same as the others :-)

Reply:
Go chaincode's fabric-ccenv and fabric-baseos images are built and published alongside fabric in this repository, might as well use the most recent images with latest Go.